doc/nrf/protocols/matter/end_product/certification.rst

```diff
@@ -288,12 +288,32 @@ You can visit the following pages to check the Bluetooth QDIDs and Thread CIDs v
 
 * `nRF5340 DK Compatibility Matrix`_
 * `nRF52840 DK Compatibility Matrix`_
+* `nRF54L15 DK Compatibility Matrix`_
 
 Matter certification process variants
```

Comment:
U can suggest separating the Recertification programs and Certification.
Certification is for the devices which are certify and different programs reduce the testing/paperwork work to get certification for different device.
While the recertification can be only utilize while the device is already Certified

Reply:
Tried to separate them.